src/main/scala/org/kframework/kore/Default.scala

```diff
@@ -0,0 +1,122 @@
+package org.kframework.kore
+
+object implementation {
```

Comment:
"default" sounded better. "implementation" is move vague -- there are many other implementations. Also, you haven't replied to my comment on making the implementation classes private.